ykpack/src/types.rs

```diff
@@ -198,7 +199,7 @@ impl Display for UnsignedIntTy {
 #[derive(Serialize, Deserialize, PartialEq, Eq, Debug, Clone, Hash)]
 pub struct Fields {
     /// Field offsets.
-    pub offsets: Vec<u64>,
+    pub offsets: Vec<OffT>,
```

Comment:
I think this is probably a good thing, but notice here we've change a `u64` into a `OffT`, when mostly we've change `u32` intos `OffT`. But I wanted to point it out explicitly!

Reply:
Yes, this was deliberate. I've used try_from and try_into where the cast could fail, so we will get crashes if things go wrong.

Reply:
Just to be clear: we never use `as`, right?

Reply:
Not for offsets, no.

(We use `as` in some places in the codegen where keeping the underlying bit pattern is correct and desired).